jsonschema-maven-plugin/src/main/java/com/github/victools/jsonschema/plugin/maven/SchemaGeneratorMojo.java

```diff
@@ -621,14 +622,15 @@ private void addJacksonModule(SchemaGeneratorConfigBuilder configBuilder, Genera
     /**
      * Write generated schema to a file.
      *
+     * @param mapper ObjectMapper to use when writing the given schema to the given file
      * @param jsonSchema Generated schema to be written
      * @param file The file to write to
      * @throws MojoExecutionException In case of problems when writing the targeted file
      */
-    private void writeToFile(JsonNode jsonSchema, File file) throws MojoExecutionException {
+    private void writeToFile(ObjectMapper mapper, JsonNode jsonSchema, File file) throws MojoExecutionException {
```

Comment:
**suggestion**: As per the above comment, there is no real need for the `ObjectMapper` to be passed in as an extra parameter here, since we're getting it from the generally available generator object now.
```suggestion
    private void writeToFile(JsonNode jsonSchema, File file) throws MojoExecutionException {
        ObjectMapper mapper = getGenerator().getConfig().getObjectMapper();
```